docs/specification/well-known-schema-formats.md

```diff
@@ -0,0 +1,33 @@
+## Well-known schema formats
```

Comment:
Does this file have no connection to "well-known encodings"? For example don't we want to specify that a protobuf schema goes along with the protobuf message encoding?

Reply:
message encoding and schema_formats are separate. As currently written it is valid to use a protobuf message encoding with a ros1msg schema format.

Reply:
If you choose to do that is there something that specifies how you should be translating between protobuf primitives and ros primitives?